### PR TITLE
User Story #50: Enter my Own Drawing Prompt

### PR DESCRIPTION
**Pull request by Irfaan-Mia**
This pull request addresses User Story #50, which allows for the core game mechanics to be completed as the first player is provided a prompt window to enter the drawing prompt they would like. This is continued by the core gameplay of broken telephone. Additional functionality such as how the game mechanics will be dealt with when a player disconnects will be done in User Story #54 including the functionality required by User Story #54.

Note:

Tests will be completed in a future sprint once functionality is completed for the game mechanics as certain functions may change. 
Please review the changes and provide any feedback or suggestions for improvement.

### CODE REVIEW

File: public/scripts/classes/client.js
Comment (MRaeesD) [inline public/scripts/classes/client.js:1]: Inconsistency in naming of button IDs

Comment (Irfaan-Mia) [reply public/scripts/classes/client.js:1]: I have attended to it thank you for letting me know. The button for submitting the drawing was named incorrectly. Please inform if it is fixed

**Review by MRaeesD — COMMENTED** [review]
The main focus of this user story has been achieved. A user can input a prompt, and the next player is able to see it after it has been submitted. The player order has been randomised. Game correctly behaves if a user leaves when it is not their turn.

Testing will be completed after more functionality is added, accounting for functions that will change later in this sprint.

Observations (for future consideration): 
Drawing does not submit when timer runs out.
If a player leaves while they are prompting or drawing, other users get stuck on the waiting screen.
Game can be started with less than 3 people.

**Comment by Irfaan-Mia** [comment]
Thank you for the consideration as stated the leaving of players will be completed in User Story #54 . The drawing timer will be attended to in the User Story #54. I looked into why the game can be started with less players seems in the User Story #49  it was commented out will ensure in the User Story #54 it is not.

**Pull request merged**